bin/redshift/run_mvn_tests_on_aws.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+# Runs Redshift connector tests.
+#
+# Run setup_aws_redshift.sh as a prerequisite for creating the Redshift instance and viewing
+# required environment variables.
+#
+# Usage:
+# run_mvn_tests_on_aws.sh
+
+set -xeuo pipefail
+
+ROOT_DIR="$(git rev-parse --show-toplevel)"
+cd "$ROOT_DIR"
+
+trap terminate INT TERM EXIT
```

Comment:
where is terminate defined?

Reply:
I missed adding it while creating the script. Added it now.